handler/graphql.go

```diff
@@ -241,6 +243,14 @@ func CacheSize(size int) Option {
 
 const DefaultCacheSize = 1000
 
+// WebsocketKeepAliveDuration allows you to reconfigure the keepAlive behavior.
+// By default, keep-alive is disabled.
```

Comment:
should we enable this by default and provide a way to opt out?

Reply:
👍 I've enabled it by default in https://github.com/99designs/gqlgen/pull/529/commits/b2f6b845c7ce9614bbde43206b0bbbb9edc4c6f3 with a ping interval of 25s (matching the expected default interval of 30s in our client https://github.com/apollographql/subscriptions-transport-ws/) if this seems reasonable?